packages/backend-modules/search/lib/inserts/questionnairesubmission.js

```diff
@@ -32,11 +32,15 @@ async function transform(row) {
             )) ||
           answer.payload?.value
 
+        const length =
+          type === 'Text' && typeValue?.length ? typeValue.length : undefined
```

Comment:
Maybe too early to comment here but beware that length is not necessarily the correct character count. Not perfect but probably better to use `[...typeValue].length` (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length#description).

Also, if the length is `0` this will be undefined. Intentional?

Reply:
Nevermind, this is not actually about string length 🤦

Reply:
(There is no such thing as too early.)

`undefined` when length 0 is intentional.